docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -44,6 +44,14 @@ Note that `createMuiTheme` accepts any number of arguments.
 If you are already using the [translations of the core components](/guides/localization/#locale-text), you can add `bgBG` as a new argument.
 The same import works with `XGrid` as it's an extension of `DataGrid`.
```

Comment:
Just to avoid breaking the line in the the period sign.

<img width="200" src="https://user-images.githubusercontent.com/42154031/122558975-17f8bd00-d015-11eb-9f11-b989ddffb1be.png" alt="image" style="max-width:100%;">


```suggestion
The same import works for `XGrid` as it's an extension of `DataGrid`.
```